src/client_state.js

```diff
@@ -8,11 +8,29 @@ export class ClientState {
 		// show the status for a certain user...
 		// ie online, offline etc
 		this.users = Immutable({});
+		// store which channels contain references to the specified user...
+		this.userChannelReferences = {};
+	}
+
+	updateUsers(users) {
+		for (const user of users) {
```

Comment:
bit more consistent to re-use updateUser here (the user null check)

```users.map(u => this.updateUser(u)```

Reply:
done, but.... i dont get why we get null users, is this a backend bug?